services/company_budget/app/src/Domain/ValueObject/MonthMax.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace App\Domain\ValueObject;
+
+class MonthMax {
+  private String $month_2;
+  
+  public function __construct(string $month) {
+    $this->month_2 = $month;
+    $this->ensureNameOnlyContainsNumbers($month_2);
+  }
+
+  private function ensureNameOnlyContainsNumbers(string $value): void {
```

Comment:
### Change Required
MonthMax is a **number** value object, **not string**. It will contain the maximum amount per month that an user can spend.

If you have any doubt, please send DM.
You could also refer to Test API doc and analyze the test api response.